src/messageParser.ts

```diff
@@ -195,7 +196,7 @@ export class MessageList {
 	}
 
 	getIndent(add?: number): string {
-		return this.indentationCharacter.repeat((this.indentation ?? 0) + (add ?? 0));
+		return this.indentationCharacter.repeat((this.indentation ?? 0) * (add ?? 1));
```

Comment:
I'm not sure I understand why this is `*`, but perhaps that's because I don't know what `this.indentation` or `add` are supposed to represent. There may be value in adding some comments so this is clearer (it seemed to me that doing `* add` could convert existing indentation of `6` and wanting `2` more to `12` instead of `8`?).

(should this affect the existing tests?)

Reply:
(It doesn't affect tests as they were using tabs as indentation characters; were the old and new values are the same in this case)

Reply:
Thanks - it's perfectly clear now :-)

(there may be value in adding a test using spaces so this is covered?)